pages/common/nikto.md

```diff
@@ -0,0 +1,27 @@
+# nikto
+
+> Web server scanner which performs comprehensive tests against web servers for multiple items, including over 6700 potentially dangerous files/programs.
+
+- To update to the latest plugins and databases:
+
+`perl nikto.pl -update`
+
+- The most basic Nikto scan requires simply a host to target:
+
+`perl nikto.pl -h 192.168.0.1`
+
+- To check on a different port:
```

Comment:
Perhaps `Specify the port number when performing a basic scan` would read better?